app/models/dtos.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+from pydantic import BaseModel
+from datetime import datetime, timezone
+
+
+class LLMModel(str, Enum):
+    GPT35_TURBO = "gpt-3.5-turbo"
+
+
+class ContentType(str, Enum):
+    TEXT = "text"
+
+
+class Content(BaseModel):
+    textContent: str
+    type: ContentType
+
+
+class SendMessageRequest(BaseModel):
+    class Template(BaseModel):
+        templateId: int
+        template: str
```

Comment:
This also does not match the Artemis implementation
```suggestion
        id: int
        content: str
```